backends/p4tools/modules/testgen/core/small_step/expr_stepper.h

```diff
@@ -98,10 +239,7 @@ class ExprStepper : public AbstractStepper {
     /// @param state the state in which the call is being made, with the call at the top of the
     ///     current continuation body.
     /// TODO(fruffy): Move this call out of the expression stepper. The location is confusing.
-    virtual void evalInternalExternMethodCall(const IR::MethodCallExpression *call,
-                                              const IR::Expression *receiver, IR::ID name,
-                                              const IR::Vector<IR::Argument> *args,
-                                              const ExecutionState &state);
+    void evalInternalExternMethodCall(const ExternInfo &externInfo);
```

Comment:
Why is this one no longer virtual?

Reply:
There is no need to make this function virtual. If an extension desires to implement their own internal externs they should be implemented in the extension.